tilt-provider.json

```diff
@@ -2,10 +2,9 @@
     "name": "metal3-bmo",
     "config": {
         "kustomize_config": false,
-        "go_main": "cmd/manager/main.go",
```

Comment:
maybe change it to main.go only ?

Reply:
I think we should not set "go_main" at all. For example if you are compare it to tilt-provider.json of [capm3](https://github.com/metal3-io/cluster-api-provider-metal3/blob/master/tilt-provider.json) or any other providers, they are not normally define "go_main". And it is not required in [CAPI book tilt-provider.json](https://cluster-api.sigs.k8s.io/developer/tilt.html#tilt-providerjson)

Reply:
ok